fe/fe-core/src/main/java/com/starrocks/catalog/CatalogRecycleBin.java

```diff
@@ -77,7 +78,12 @@ public class CatalogRecycleBin extends MasterDaemon implements Writable {
     private com.google.common.collect.Table<Long, String, RecycleTableInfo> nameToTableInfo;
     private Map<Long, RecyclePartitionInfo> idToPartition;
 
-    private Map<Long, Long> idToRecycleTime;
+    // if enable erase later, the real recycle time is RecycleTime + LATE_RECYCLE_INTERVAL_SECONDS
+    // This is only useful on master, when tablet scheduler repair a tablet that is about to expire
+    // We can make the db/table/partition infomation stay longer until the asynchronized agent task finish.
+    protected static int LATE_RECYCLE_INTERVAL_SECONDS = 60;
```

Comment:
What's  the purpose of the LATE_RECYCLE_INTERVAL_SECONDS?

Reply:
Added some comments to explain why.